packages/protocol/contracts/stability/SortedOracles.sol

```diff
@@ -179,9 +217,33 @@ contract SortedOracles is ISortedOracles, Ownable, Initializable {
       address(0)
     );
     emit OracleReported(token, msg.sender, now, value, DENOMINATOR);
+    recomputeRate(token);
+  }
+
+  function recomputeRate(address token) internal {
+    uint256 originalMedian = limitedMedianRate[token];
     uint256 newMedian = rates[token].getMedianValue();
+    if (limitedMedianRateTimestamp[token] != 0 && maxMedianChangeRatePerDay.unwrap() != 0) {
+      uint256 td = now.sub(limitedMedianRateTimestamp[token]);
+      if (td > 1 days) td = 1 days;
```

Comment:
Why cap `td` at one day?

Reply:
I think it's good idea to have some upper limit for the change. But I don't know how often they are supposed to be updated.

Reply:
I'm not totally sure I agree this is necessary, It seems to me that a limit on the rate of change per unit time is sufficient